CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ set(CMAKE_CXX_STANDARD 14)
 set(IREE_IDE_FOLDER IREE)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+# NOTE: new options added here should also be added to iree_cross_compile.cmake.
```

Comment:
Can you use LINT.IfChange? That's our standard way of annotating this, which has internal tooling support.

Reply:
Ah right. Thanks for the reminder! :)